src/codeu/chat/util/UptimeServerInfo.java

```diff
@@ -0,0 +1,28 @@
+package codeu.chat.util;
+
+// why is this class in the util package? Can we combine the two ServerInfo classes in the
+// codeu.chat.common package?
+
+import java.io.IOException;
+
+public final class UptimeServerInfo {
+
+    public Time startTime;
+    private final static String SERVER_VERSION = "1.0.0";
+    public Uuid version;
+
+    public UptimeServerInfo() throws IOException{
+        //Set the startTime equal to the Time value that the server was started
+        this.startTime = Time.now();
+        this.version = Uuid.parse(SERVER_VERSION);
+    }
+
+    public UptimeServerInfo(Uuid version) throws IOException{
```

Comment:
This UpTimeServerInfo constructor does not carry a Time value, which I think it needs to. This constructor needs a startTime, maybe add:    this.startTime = Time.now();